addons/website/static/src/snippets/s_facebook_page/options.js

```diff
@@ -171,6 +199,20 @@ options.registry.facebookPage = options.Class.extend({
                         type: "warning",
                     });
                 }
+            }, () => {
+                const fbPageElement = this._getFbPageElement();
```

Comment:
To test this, what I did, I go to edit mode, disconnected from internet and then I click on facebook page snippet, so we will reach to this code but we already have facebook page rendered so this will not go to below if condition and snippet will be blank and no warning is displayed, as I said we do not have fallback message on snippet in case of connection failure.